fe1-web/network/NetworkManager.ts

```diff
@@ -3,6 +3,8 @@ import { NetworkConnection } from './NetworkConnection';
 import { NetworkError } from './NetworkError';
 import { defaultRpcHandler, JsonRpcHandler } from './RpcHandler';
 
+const defPort = 9000;
```

Comment:
unclear what it means. I read it first as "DefinitionPort".

Reply:
removed in a later commit